### ISSUE TITLE
Myq Homebridge plug not working after latest update

### ISSUE THREAD

**Issue by Jrickw1** [hjdhjd/homebridge-myq#522]
**Describe The Problem:**
can no longer control garage door with HomeKit after last myq plug-in update. It just doesn’t work. HomeKit can see the accessory and thinks the garage door is opening when I tell it to but nothing happens. 

**To Reproduce:**
not sure there are any steps other that what I describe above. It just doesn’t work. I’ve tried rebooting the hub but still nothing. Other Homebridge accessories are functioning normally. 

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.

9/25/2021, 2:10:26 PMmyQ BridgemyQERRORrequest to https://partner-identity.myq-cloud.com/connect/authorize?client_id=IOS_CGI_MYQ&code_challenge=ilMSrMYJNcL4j40rbvcOC4cZgq81EdwON_kAWlVW7Ck&code_challenge_method=S256&redirect_uri=com.myqops%3A%2F%2Fios&response_type=code&scope=MyQ_Residential+offline_access failed, reason: write EPROTO 1995528656:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:
9/25/2021, 2:10:26 PMmyQ BridgemyQERRORmyQ API: Unable to access the OAuth authorization endpoint.
9/25/2021, 2:12:26 PMmyQ BridgemyQERRORrequest to https://partner-identity.myq-cloud.com/connect/authorize?client_id=IOS_CGI_MYQ&code_challenge=WJkAKg-U65Hhv6_wdh_urAUE5ix812PYzg8SkdEqSOQ&code_challenge_method=S256&redirect_uri=com.myqops%3A%2F%2Fios&response_type=code&scope=MyQ_Residential+offline_access failed, reason: write EPROTO 1995528656:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:
9/25/2021, 2:12:26 PMmyQ BridgemyQERRORmyQ API: Unable to access the OAuth authorization endpoint.
9/25/2021, 2:14:26 PMmyQ BridgemyQERRORrequest to https://partner-identity.myq-cloud.com/connect/authorize?client_id=IOS_CGI_MYQ&code_challenge=GWWtt3YAblsl0SbxCwz5RMdpSjHBHMES9zO4Aa02N44&code_challenge_method=S256&redirect_uri=com.myqops%3A%2F%2Fios&response_type=code&scope=MyQ_Residential+offline_access failed, reason: write EPROTO 1995528656:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332:
9/25/2021, 2:14:26 PMmyQ BridgemyQERRORmyQ API: Unable to access the OAuth authorization endpoint.

```

**Homebridge Configuration:**

```json
Show the relevant portion of your homebridge config.json here, if needed.
Remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **Homebridge Version:**  4.0.100
* **Node Version:** 
* **Homebridge-myQ Plugin Version**:2.7.1
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:**iPad and iPhone running iOS 15 and iPad so 15
* **Operating System and OS Version:**

**Comment by Jrickw1** [comment]
Update on this. The issue may be my eero network. I discovered that I cannot use the myQ iOS app when connected to my home WiFi network - I get an error reaching server message. If I disconnect from WiFi the app launches fine. Since my home bridge is on the same eero network I’m thinking that it’s preventing the myQ homebridge plug-in from connecting. This only started happening within the last few days.

**Comment by johnshumate** [comment]
I’m on eero at home too and thought similar when my MyQ app wouldn’t work on my Wi-Fi but would work on AT&T cellular. I turned on a separate Wi-Fi from my Xfinity modem and MyQ still wouldn’t connect, so I’m not blaming eero right now.

**Comment by Jrickw1** [comment]
For me this issue has been resolved. I went into the Eero app / Discover menu / HomeKit — and the app reported that HomeKit permission was not enabled under settings. Even though this was not correct, I went into the iOS Settings App / Eero / and toggled HomeKit permission (which was already on) to off, force quite the Eero app, and toggled HomeKit permission back on and now everything works. I can’t say for certain whether this “fix” had anything to do with why everything works now or whether it’s because Chamberlain fixed something on their end and the timing is just a coincidence. (Also can’t say why johnshumate was able to replicate the same issue with an XFinity router — although I’ll note XFinity is also our service provider.) Either way HomeBridge MyQ functionality seems to be restored.
